packages/@aws-cdk/cloudformation-include/README.md

```diff
@@ -124,6 +124,24 @@ and any changes you make to it will be reflected in the resulting template:
 condition.expression = core.Fn.conditionEquals(1, 2);
 ```
 
+## Outputs
+
+If your template uses [CloudFormation Outputs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html),
+you can retrieve them from your template:
+
+```typescript
+import * as core from '@aws-cdk/core';
+
+const output: core.CfnOutput = cfnTemplate.getOutput('MyOutput');
+```
+
+The `CfnOutput` object is mutable,
+and any changes you make to it will be reflected in the resulting template:
+
+```typescript
+output.value = "some new value");
```

Comment:
1. Unnecessary `)` at the end.
2. Can we make the example a little more realistic? For example, `output.value = cfnBucket.attrArn`.